harvard-deakin-university.csl

```diff
@@ -0,0 +1,214 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>Harvard - Deakin University </title>
+    <id>http://www.zotero.org/styles/harvard-deakin-university</id>
+    <link href="http://www.zotero.org/styles/harvard-deakin-university" rel="self"/>
+    <link href="http://www.deakin.edu.au/students/studying/study-support/referencing/harvard" rel="documentation"/>
+        <category citation-format="author-date"/>
+    <category field="generic-base"/>
+    <summary>Style for Deakin University </summary>
+    <updated>2016-09-15T11:36:24+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+    <terms>
```

Comment:
yeah, we've become more stringent on reviewing new style submissions. There's not much you can break here. Just delete all terms except the ones mentioned.